action/action-router.js

```diff
@@ -0,0 +1,36 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+
+const knexConfig = {
```

Comment:
This is fine, though we already have it in our knexfile.js.. we can take advantage of the fact and create a dbConfig.js file with the following content:

```
const knex = require('knex');

const knexConfig = require('../knexfile.js');

module.exports = knex(knexConfig.development);
```

Then we can simply import `const db = require("../data/dbConfig.js");` in this router file or in your model file (not required for this project).